pay-api/src/pay_api/services/direct_pay_service.py

```diff
@@ -234,19 +236,28 @@ def _get_refund_token(cls):
         return token_response
 
     @staticmethod
-    def _build_automated_refund_payload(invoice: InvoiceModel):
+    def _build_automated_refund_payload(invoice: InvoiceModel, refund_revenue: List[Dict]):
         """Build payload to create a refund in PAYBC."""
         receipt = ReceiptModel.find_by_invoice_id_and_receipt_number(invoice_id=invoice.id)
         invoice_reference = InvoiceReferenceModel.find_by_invoice_id_and_status(
             invoice.id, InvoiceReferenceStatus.COMPLETED.value)
-        # Future: Partial refund support - This is backwards compatible
-        # refundRevenue: [{
-        #        'lineNumber': 1,
-        #        'refundAmount': 50.00,
-        #    }]
+
+        refund_details = []
+        for line in refund_revenue:
```

Comment:
Don't worry about this part, I'll handle it is part of the PAYBC ticket. 

I think you need to look at what happens before, invoice status, etc

Reply:
So we might want to store the payload or a representation of the partial refund request. Otherwise all we have to work with right now is the refund amount, but that doesn't really explain line by line where the refund was coming from.

Reply:
Payment line item table, doesn't have any field for `refund` and no concept of payment status